### PR TITLE
feat: Add Splash Screen

### PR DESCRIPTION
**Pull request by S2606**
Fixes https://github.com/fossasia/susi_android/issues/895

Screenshot
screenshot_20170818-073014

### CODE REVIEW

**Comment by betterclever** [comment]
@S2606 Good work. Please also add relevant screenshots in the PR. Thanks :+1:

**Comment by S2606** [comment]
@chiragw15 Please review

**Comment by chiragw15** [comment]
Please take out "Copyright notice". No need here.

File: app/src/main/res/layout/activity_splash.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    tools:context="org.fossasia.susi.ai.intro.SplashScreen">
+
+    <ImageView
+        android:id="@+id/imgLogo"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_centerInParent="true"
+        app:srcCompat="@drawable/ic_susi_blue" />
+
+    <TextView
```
Comment (chiragw15) [inline app/src/main/res/layout/activity_splash.xml]: remove this textview

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <resources>
     <!--General Strings-->
     <string name="app_name" translatable="false">Susi</string>
+    <string name="copyright" translatable="false">Copyright &#169; 2017 FOSSAsia</string>
```
Comment (chiragw15) [inline app/src/main/res/values/strings.xml]: remove this

**Comment by S2606** [comment]
changes made @chiragw15

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#Tue May 02 22:30:08 IST 2017
+#Thu Aug 17 12:21:41 IST 2017
```
Comment (amitiwary999) [inline gradle/wrapper/gradle-wrapper.properties]: Not needed this change. You have not made any change in gradle properties.

Comment (S2606) [inline gradle/wrapper/gradle-wrapper.properties]: updated @amitiwary999

**Comment by mariobehling** [comment]
We deleted the splash screen earlier, because we want the user to get right to the feature.

**Pull request closed without merging**